app/routes/admin/events/list.js

```diff
@@ -117,10 +117,10 @@ export default class extends Route.extend(EmberTableRouteMixin) {
     }
 
     filterOptions = this.applySearchFilters(filterOptions, params, searchField);
-
+    
     let queryString = {
       get_trashed    : true,
-      include        : 'tickets,sessions,speakers,owner,organizers,coorganizers,track-organizers,registrars,moderators',
+      include        : 'general-statistics,owner,organizers,coorganizers,track-organizers,registrars,moderators',
```

Comment:
@iamareebjamal You will need to make this change in `routes/events/list` too

Reply:
Also, Is the `general-statistics` needed ?

Reply:
> You will need to make this change in routes/events/list too

Why?

> Also, Is the general-statistics needed ?

Yes, sessions and speaker counts are shown using general statistics

Reply:
> Yes, sessions and speaker counts are shown using general statistics

Speaker and Session Statistics are getting shown without using the `general-statistics' ?